<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>19</y>
     <width>611</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="mimeTypeLabel">
      <property name="text">
       <string>Mime Type</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="clipIdLabel">
      <property name="text">
       <string>Clip ID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="clipIdInput"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="userIdInput"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="userNameInput"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="userNameLabel">
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="userIdLabel">
      <property name="text">
       <string>User ID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="mimeTypeInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Results:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="searchListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>611</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
